lib/rubocop/cop/rspec/variable_name.rb

```diff
@@ -14,6 +17,14 @@ module RSpec
       #   let(:user_name) { 'Adam' }
       #   subject(:user_name) { 'Adam' }
       #
+      #   # okay with `userFood` listed in `IgnoredVariables`
```

Comment:
Typically we describe it in a separate `@example`:
```ruby
      # @example `Prefixes` configuration
      #
      #   # .rubocop.yml
      #   # RSpec/ContextWording:
      #   #   Prefixes:
      #   #     - when
      #   #     - with
      #   #     - without
      #   #     - if
      #   #     - unless
      #   #     - for
      #
      # @example
      #   # bad
      #   context 'the display name not present' do
      #     # ...
      #   end
      #
      #   # good
      #   context 'when the display name is not present' do
      #     # ...
      #   end
```

do you mind adding two examples to demonstrate how we expect the configuration of those options?
Otherwise, it may create a false impression that `IgnoredVariables`/`IgnoredPatterns` only relate to `EnforcedStyle: snake_case`.